source/Command.h

```diff
@@ -28,6 +29,12 @@ class DataNode;
 // A single Command object can represent multiple individual commands, e.g.
 // everything the AI wants a ship to do, or all keys the player is holding down.
 class Command {
+private:
+	// This must match the maximum command number in Command.cpp or the build will fail.
+	static const unsigned COMMAND_COUNT = 33;
```

Comment:
You don't need this variable if you `.size()` everywhere

Reply:
it is gone